config/routes.rb

```diff
@@ -18,6 +18,7 @@
       default: true) do
       resources :users, only: [:show, :update, :index]
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/AlignHash
**Severity**: INFO
**File**: config/routes.rb L19

Align the elements of a hash literal if they span more than one line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*